### PR TITLE
Order

### PR DESCRIPTION
**Pull request by ilkaypolat1985**

### CODE REVIEW

File: petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductRemoveRequest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class OrderProductRemoveRequest {
+    @NotNull
+    private Long userId;
+
+    @NotNull
+    private Long orderId;
+
```
Comment (aliturgutbozkurt) [inline petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductRemoveRequest.java:18]: Client shouldn't know order

File: petshop-be/domain/src/main/java/com/turkninja/petshop/api/response/order/OrderItemGetResponse.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.*;
+
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class OrderItemGetResponse {
+    private OrderEntity order;
```
Comment (aliturgutbozkurt) [inline petshop-be/domain/src/main/java/com/turkninja/petshop/api/response/order/OrderItemGetResponse.java:14]: May couse infinite loop

File: petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductAddRequest.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class OrderProductAddRequest {
+    @NotNull
+    private Long userId;
+
+    private Long orderId;
+
```
[the remark this answers is not in the document]
Comment (ilkaypolat1985) [reply petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductAddRequest.java]: > Client shouldn't know about order

Fixed

File: petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductRemoveRequest.java (hunk 1 above)
Comment (ilkaypolat1985) [reply petshop-be/domain/src/main/java/com/turkninja/petshop/api/request/order/OrderProductRemoveRequest.java]: > Client shouldn't know order

Fixed

File: petshop-be/domain/src/main/java/com/turkninja/petshop/api/response/order/OrderItemGetResponse.java (hunk 2 above)
Comment (ilkaypolat1985) [reply petshop-be/domain/src/main/java/com/turkninja/petshop/api/response/order/OrderItemGetResponse.java]: > May couse infinite loop

Fixed

**Pull request merged**